### PR TITLE
Chore: small Admin documentation improvements

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Chore: small grammar and spelling improvements

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -1603,7 +1603,7 @@ The following table shows the default value and the allowed ones:
 |PASSWORDLOCKTIME | 60 | a positive integer value greater than or equal to 60; the unit of time is minutes
 |===
 
-The value of PASSWORDLOCKTIME is the number of minutes that an account is locked after failing to specify the correct password the number of times specified by
+The value of PASSWORDLOCKTIME is the number of minutes that an account is locked after failing to specify the correct password and the number of times specified by
```
Comment (mizzioisf) [inline doc_admin/AdminManual.adoc]: I would say "for" the number of times.

**Review by mizzioisf — COMMENTED** [review]
Nice job ! I have left just one single comment, besides that everything looks great :-)

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (dbmalkovsky) [reply doc_admin/AdminManual.adoc]: Done; thanks

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**